mingw-w64-clang/update-clang-from-msys2.sh

```diff
@@ -49,7 +49,8 @@ mv upstream/$pkgname/README-patches.md ./ || die "$0: failed to replace existing
 
 sed -e "s/pkgrel=[.0-9]\+\(.*\)/pkgrel=$new_pkgrel\1/" \
     -e 's/-DCMAKE_BUILD_TYPE=Release/-DCMAKE_BUILD_TYPE=MinSizeRel/' \
-    -e 's/-DLLVM_TARGETS_TO_BUILD=.*/-DLLVM_TARGETS_TO_BUILD=Native/' \
+    -e 's/-DLLVM_TARGETS_TO_BUILD=".*"/-DLLVM_TARGETS_TO_BUILD=Native/' \
+    -e 's/-DLLVM_TARGETS_TO_BUILD=[A-Za-z0-9;]/-DLLVM_TARGETS_TO_BUILD=Native/' \
```

Comment:
I think we need to match more than a single letter of the `LLVM_TARGETS_TO_BUILD` value:

```suggestion
    -e 's/-DLLVM_TARGETS_TO_BUILD=".*"/-DLLVM_TARGETS_TO_BUILD=Native/' \
    -e 's/-DLLVM_TARGETS_TO_BUILD=[A-Za-z0-9;]*/-DLLVM_TARGETS_TO_BUILD=Native/' \
```

Reply:
Right. That '*' somehow got lost during testing.